app.py

```
@@ -7,6 +7,9 @@
 from sklearn.preprocessing import LabelEncoder
 import os
 from pathlib import Path
+from datetime import datetime
+import base64
+from io import BytesIO
 
 # Import our custom modules
 from config import *
```

Ruff (0.12.2)

`from config import *` used; unable to detect undefined names

(F403)